<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>4</number>
   </property>
   <item row="0" column="0" rowspan="3">
    <widget class="QGroupBox" name="">
     <property name="title">
      <string>Global Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If set to &amp;quot;On&amp;quot; the individual values for threshold, pz, gain and shaping time are used.&lt;/p&gt;&lt;p&gt;If set to &amp;quot;Off&amp;quot; the values from the &amp;quot;Common&amp;quot; columns are used.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Single Channel Mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="rb_single_channel_on">
           <property name="text">
            <string>On</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_single_channel_off">
           <property name="text">
            <string>Off</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string>Baseline restorer</string>
        </property>
        <property name="text">
         <string>BLR</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="rb_blr_on">
           <property name="text">
            <string>On</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_blr_off">
           <property name="text">
            <string>Off</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="toolTip">
         <string>Baseline restorer threshold</string>
        </property>
        <property name="text">
         <string>BLR threshold</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spin_blr_threshold">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Multiplicity hi</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spin_multiplicity_high">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Multiplicity lo</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spin_multiplicity_low">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Threshold offset</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spin_threshold_offset">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Shaper offset</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="spin_shaper_offset">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Coincidence time</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="spin_coincidence_time">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="">
     <property name="title">
      <string>Auto PZ</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Auto PZ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="combo_auto_pz"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Current channel:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_current_auto_pz_channel">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="gb_copy">
     <property name="title">
      <string>Copy Function</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QPushButton" name="pb_copy_common2single">
        <property name="text">
         <string>RC Common → Single</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pb_copy_rc2panel">
        <property name="text">
         <string>RC → Panel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pb_copy_panel2rc">
        <property name="text">
         <string>Panel → RC</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="minimumSize">
         <size>
          <width>131</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Version:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_version">
        <property name="text">
         <string>unknown</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Channel Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="combo_channel"/>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QStackedWidget" name="stacked_channels"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>195</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>spin_blr_threshold</zorder>
  <zorder>combo_monitor_channel</zorder>
  <zorder>label_7</zorder>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>label</zorder>
  <zorder>spin_shaper_offset</zorder>
  <zorder>spin_threshold_offset</zorder>
  <zorder>label_5</zorder>
  <zorder>spin_multiplicity_high</zorder>
  <zorder>label_9</zorder>
  <zorder>spin_multiplicity_low</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>spin_coincidence_time</zorder>
  <zorder>le_currAutoPZChan</zorder>
  <zorder>spin_pz_display_range</zorder>
  <zorder>label_12</zorder>
  <zorder>label_11</zorder>
  <zorder>le_version</zorder>
  <zorder>label_10</zorder>
  <zorder>label_8</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pb_copy_rc2panel</zorder>
  <zorder>pb_copy_panel2rc</zorder>
  <zorder>pb_copy_common2single</zorder>
  <zorder>pb_copy_panel2rc</zorder>
  <zorder></zorder>
  <zorder>groupBox</zorder>
  <zorder>horizontalSpacer_2</zorder>
 </widget>
 <tabstops>
  <tabstop>rb_single_channel_on</tabstop>
  <tabstop>rb_single_channel_off</tabstop>
  <tabstop>rb_blr_on</tabstop>
  <tabstop>rb_blr_off</tabstop>
  <tabstop>spin_blr_threshold</tabstop>
  <tabstop>spin_multiplicity_high</tabstop>
  <tabstop>spin_multiplicity_low</tabstop>
  <tabstop>spin_threshold_offset</tabstop>
  <tabstop>spin_shaper_offset</tabstop>
  <tabstop>spin_coincidence_time</tabstop>
  <tabstop>combo_auto_pz</tabstop>
  <tabstop>pb_copy_panel2rc</tabstop>
  <tabstop>pb_copy_rc2panel</tabstop>
  <tabstop>pb_copy_common2single</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>combo_channel</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stacked_channels</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
